### ISSUE TITLE
[Bug]: Chrome config additional profiles may have "Cookies" filename instead of "Cookie"

### ISSUE THREAD

**Issue by InstanceGaming** [thewh1teagle/rookie#9]
### What happened?

In `config.rs`, the Chrome config `data_paths` list is lacking additional patterns,
```rust
"%LOCALAPPDATA%/Google/Chrome{channel}/User Data/Profile */Network/Cookies", /*notice the s at the end*/
...
"%APPDATA%/Google/Chrome{channel}/User Data/Profile */Network/Cookies"
```

I have witnessed first-hand this file being created instead of `Cookie` for Chrome 118.0.5993.71 with a signed-in google account attached. I would consider this a bug over a feature request because all the rest of the Chromium-based browser configs have it already. I can submit a PR if you want it, but the solution in this case is so trivial I don't think its worth the extra effort rather than rolling this fix into the next update.

### Steps to reproduce

1. Install Chrome 118.0.5993.71
2. Sign the browser into a new Google account
3. A new Profile directory is created, i.e. `Profile n/`
4. Check the cookie database name, `Cookies` instead of `Cookie` under `Profile n/Network/`.

### What browsers are you seeing the problem on?

Chrome

**Comment by thewh1teagle** [comment]
You are correct; it's a typo. 
Good catch! 
Thank you very much for reporting it. I'll fix it and will update you here when it's fixed.

**Comment by thewh1teagle** [comment]
fixed in 3.0.1
